config/xinitrc

```diff
@@ -12,19 +12,20 @@ export VX_METADATA_ROOT="/vx/code"
 
 # Elo screen, adjust orientation and corresponding touch coordinates
 # xinput is not ready to be used until X has fully started, so we have to delay
-ELO="Elo Touch ET1502L"
-(sleep 4 && if xinput list | grep "$ELO" > /dev/null; then
+ELO="Elo Touch"
+(sleep 4 && if xinput list | grep -i "$ELO" > /dev/null; then
```

Comment:
Added this `-i` (for case insensitivity) since I noticed it in https://github.com/votingworks/vxsuite-complete-system/pull/313. May not be necessary but am playing it safe